docs/releases/v2023.2.rst

```diff
@@ -0,0 +1,272 @@
+Gluon 2023.2
+============
+
+Important notes
+---------------
+
+Upgrades to v2023.2 and later releases are only supported from releases v2022.1 and later.
+This is due to migrations that have been removed to simplify maintenance.
+
+
+Deprecation of Tunneldigger VPN
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tunneldigger is set to be removed from core in the next Gluon release. It is recommended
+to migrate to fastd or WireGuard. Alternatively, tunneldigger will be moved to the
+community-packages repository and can be installed from there.
+
+
+Site changes
+------------
+
+Image customization
+~~~~~~~~~~~~~~~~~~~
+
+`GLUON_FEATURES` as well as `GLUON_PACKAGES` have been replaced by a more flexible customization framework
+based on a Lua DSL. Package and feature selection can be specified more granularly at both target and device level.
+
+Additionally, this framework also allows communities to specify which devices should or should not be built.
+For more information, see the :ref:`documentation <site-image-customization>`.
+
+
+Added hardware support
+----------------------
+
+armsr-armv7
+~~~~~~~~~~~
+
+- ARM
+
+  - SystemReady 32-bit (EFI)
+
+
+armsr-armv8
+~~~~~~~~~~~
+
+- ARM
+
+  - SystemReady 64-bit (EFI)
+
+
+ath79-generic
+~~~~~~~~~~~~~
+
+- AVM
+
+  - FRITZ!Repeater 1750E
+
+- Sophos
+
+  - AP100
+  - AP100c
+  - AP55
+  - AP55c
+
+- TP-Link
+
+  - Archer C60 (v1)
+  - EAP225-Outdoor v3
+  - TL-WR2543N/ND (v1)
+
+
+ath79-mikrotik
+~~~~~~~~~~~~~~
+
+- MikroTik
+
+  - wAPR-2nD (wAP R)
+
+
+ipq40xx-generic
+~~~~~~~~~~~~~~~
+
+- ZTE
+
+  - MF289F
+
+
+mediatek-filogic
+~~~~~~~~~~~~~~~~
+
+- ASUS
+
+  - TUF-AX4200
+
+- Cudy
+
+  - WR3000 (v1)
+
+- GL.iNet
+
+  - GL-MT3000
+
+- NETGEAR
+
+  - WAX220
+
+- Ubiquiti
+
+  - Unifi 6 Plus
+
+- ZyXEL
+
+  - NWA50AX Pro
+
+
+mpc85xx-p1010
+~~~~~~~~~~~~~
+
+- Enterasys
+
+  - WS-AP3715i
+
+
+ramips-mt7621
+~~~~~~~~~~~~~
+
+- TP-Link
+
+  - EAP615-Wall
+
+- Wavlink
+
+  - WS-WN572HP3 4G
+
+
+ramips-mt76x8
+~~~~~~~~~~~~~
+
+- ASUS
+
+  - RT-AX53U
+
+- ZyXEL
+
+  - WSM20
+
+
+Removed hardware support
+------------------------
+
+ath79-generic
+~~~~~~~~~~~~~
+
+- TP-Link
+
+  - Archer C60 (v1)
+  - RE355
+  - RE450 (v1)
+
+- Ubiquiti
+
+  - NanoBeam 5AC 19 (XC)
+  - NanoBeam M5 (XW)
+  - NanoStation Loco M2/M5 (XW)
+  - NanoStation M2/M5 (XW)
+
+
+ramips-mt7621
+~~~~~~~~~~~~~
+
+- TP-Link
+
+  - RE305
+
+
+Features
+--------
+
+TLS support
+~~~~~~~~~~~
+
+Gluon now supports TLS for libustream. Communities can include the new `tls` feature
+which automatically installs the corresponding package as well as the ca-bundle.
+
+Communities should switch to this feature instead of including the packages individually.
+Custom packages should switch to this dependency.
```

Comment:
I think this section should be reworded, as it is unclear what the TLS support refers to, and I'd argue that Gluon has always "supported" TLS.

```suggestion
Gluon now provides HTTPS client support when the `tls` feature is included in the site
configuration, allowing nodes to establish encrypted connections to autoupdater mirrors,
opkg repositories and other HTTPS servers.

Existing site configurations that add libustream TLS packages should switch to the `tls`
feature instead, which will always include the recommended TLS implementation as well
as common CA certificates (`ca-bundle`).

Communities should switch to this feature instead of including the packages individually.
Custom packages should switch to depend on the `gluon-tls` package instead of referencing
specific libustream packages directly.
```
(this is written assuming that libustream TLS is only used for HTTPS client support, are there
other uses?)

Reply:
the `should switch` sounds a bit repetitive, but i currently don't have a better wording